### DISCUSSION TITLE
Export is broken on Snap /PopOS 22.04

### DISCUSSION THREAD

**Discussion by thob** [standardnotes/forum#2897] [General]
Getting this when I want to Export. Happens on Snap v.x.x.296 /PopOS22.04. Works on the web.
![sn_export](https://user-images.githubusercontent.com/755337/201914916-1be8b00e-89f1-4376-b7ca-d3cfcfd56a29.png)

**Comment by effieeee** [comment]
Hi @thob, strange, though how about the AppImage? Do you find the same behavior while using it instead of the Snap?

**Comment by thob** [comment]
I'm using Snap for its update mechanism. This never worked nicely w/ your Appimage. But let's go back to the bug. Any chance you can figure out what's going on?

**Comment by effieeee** [comment]
I see, though yes, looking back at the issue, could it just be related to your font cache for the snap as described in [this link](https://askubuntu.com/questions/1224125/font-characters-displayed-as-squares-in-ubuntu-18-04)? (though the forum is for Ubuntu specifically)

**Comment by thob** [comment]
hey, it mentions chromium, is this what SN is using?

**Comment by thob** [comment]
it worked. in my case it's `~/snap/standard-notes/common/.cache/fontconfig`

**Comment by thob** [comment]
the issues re-surfaced. Is this a SN issue or Snap?

**Comment by effieeee** [comment]
I'd say it's more likely a Snap issue since you mentioned that the fonts display fine while exporting a backup on Web 🤔 Does the method still work when you try it again now?

**Comment by jpbaril** [comment]
I'm also having this problem.
Unless I did it wrong, deleting fontconfig cache did not help.
On freshly released Ubuntu 23.04
